.tekton/pull-request.yaml

```diff
@@ -46,6 +50,32 @@ spec:
             - name: run-tests
               image: registry.access.redhat.com/ubi8/ubi:latest
               script: cd $(workspaces.workspace.path) && ./automation/test.sh
+      - name: run-gitlint
+        params:
+          - name: git-url
+            value: $(params.git-url)
```

Comment:
This param is unused and can be removed.

Reply:
+1

Reply:
done